debian/rules

```diff
@@ -43,15 +43,15 @@ binary-syncd-rpc:
 #	-DCMAKE_LIBRARY_PATH=$(DEB_HOST_MULTIARCH)
 
 override_dh_auto_configure:
+    ifeq ($(shell cat /tmp/syncd-build), --enable-rpcserver=yes)
+	sed -i 's/ENABLE_SAITHRIFT=0/ENABLE_SAITHRIFT=1/' syncd/usr/bin/syncd_init_common.sh
+    else
+	sed -i 's/ENABLE_SAITHRIFT=1/ENABLE_SAITHRIFT=0/' syncd/usr/bin/syncd_init_common.sh
+    endif
```

Comment:
My path is outside of `debian/`. My new files are in `syncd/usr/bin/`, and `syncd/` is on the same level as `debian/`. I have tested this with our build infrastructure.